festim/boundary_conditions/fluxes/surface_kinetics.py

```diff
@@ -0,0 +1,136 @@
+from festim import FluxBC
+from fenics import *
+import sympy as sp
+
+
+class SurfaceKinetics(FluxBC):
+    """
+    FluxBC subclass allowing to include surface processes on in 1D H transport simulations.
+
+    d c_s / dt = k_bs l_abs c_m (1 - c_s/N_s) - k_sb c_s (1 - c_b/N_b) + J_vs
```

Comment:
@RemDelaporteMathurin 
The issue with API appears even if I delete everything (Args, Attributes, ..) except text description. Also, the issue appears when I compile locally

Reply:
I found that changing the subsection title (e.g. [this](https://github.com/festim-dev/FESTIM/blob/b56a3dc1dca432109d489c012aefea280698b807/docs/source/api/boundary_conditions.rst?plain=1#L31-L32)) to a higher level solves the problem

Reply:
I still don't see the classes appear: 
![image](https://github.com/festim-dev/FESTIM/assets/40028739/e2d4a91f-328d-4cf5-a8fa-3ba12fc43668)

Reply:
Right, I haven't figure out how to fix this. I kept the docstrings for `SurfaceKinetics` for now. Two things I noticed: if you fix `SievertsBC` then the Dirichlet classes also dissapear; if you delete several Flux classes fro API then `SurfaceKinetics` and other classes appear.

Reply:
Worked on in #797